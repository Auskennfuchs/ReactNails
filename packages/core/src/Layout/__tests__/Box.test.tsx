import { render } from '@testing-library/react'
import React from 'react'

import { NailsApp, SpacingType } from '../..'
import { Box, MediaQueryBreakPoint } from '..'

describe('Box tests', () => {
    test('should render', () => {
        const { container } = render(<Box space={SpacingType.m} />, {
            wrapper: ({ children }) => (
                <NailsApp>
                    {children}
                </NailsApp>
            ),
        })
        expect(container).toMatchSnapshot()
    })

    test('should render space media query', () => {
        const { container } = render(<Box space={{ [MediaQueryBreakPoint.small]: SpacingType.m }} />, {
            wrapper: ({ children }) => (
                <NailsApp>
                    {children}
                </NailsApp>
            ),
        })
        expect(container).toMatchSnapshot()
    })
})
