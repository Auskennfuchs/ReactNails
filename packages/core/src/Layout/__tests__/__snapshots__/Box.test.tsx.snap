// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box tests should render 1`] = `
<div>
  <div
    class="sc-bdvvtL gPlMRc"
  />
</div>
`;

exports[`Box tests should render space media query 1`] = `
<div>
  <div
    class="sc-bdvvtL hmqSrW"
  />
</div>
`;
